### ISSUE TITLE
Can't enter portals

### ISSUE THREAD

**Issue by UnoriginalUmbrella** [Johni0702/BetterPortals#274]
I made a portal with the Twilight forest mod, but it just appears to be a normal portal, and when I jumped in it doesn't teleport me at all. I tried making the portal again but the same thing happened. I made sure I had no incompatible mods.

**Comment by Johni0702** [comment]
Sounds like https://github.com/Johni0702/BetterPortals/issues/253 which will be fixed in the next update. For now, make sure your portal frame only has vanilla grass blocks.

**Comment by UnoriginalUmbrella** [comment]
Okay si I managed to get the portals "working. Apparently when I jump into it I get teleported underground or on the surface, where I somehow suffocate. I also get stuck because the rest of the area won't render.

**Comment by Johni0702** [comment]
You'll have to give more info that that. If I jump into a TF portal, I'm neither suffocate (disregarding #248) nor are there any rendering issues. I generally always recommend reading [this article](https://www.chiark.greenend.org.uk/~sgtatham/bugs.html) to anyone who hasn't yet.

**Comment by UnoriginalUmbrella** [comment]
Okay here's a more concise explanation. So I get the portals somewhat working, and when I first jumped in I was stuck inside a cave and couldn't move, so I had to go into spectator mode to get out. Then I realized my world was a weird amalgamation of the overworld and twilight forest. I then changed to survival and suffocated to death while I was outside. Then I respawned n the twilight forest with no portal, so I had to use journey map to escape. I then tried it again in a different are, where I got stuck in mid air, while half of the area didn't render.
